<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PRUEBA PSICOMÉTRICA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Fondo">
    <property name="geometry">
     <rect>
      <x>-200</x>
      <y>-60</y>
      <width>1231</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/cct/wallpaper_main.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="Inputs">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Graytransparent">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>371</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/cct/fondo-png-transparente.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="coordBoton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>COORDINACIÓN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refleBoton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>REFLEJOS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="predicBoton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>PREDICCIÓN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="puntajeBoton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>PUNTAJE</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPRUEBAS">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-4</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>PRUEBAS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkCOORD">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkREFLE">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkPREDI">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>◄ REGRESAR</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Condumas">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/cct/fondo-png-transparente.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#000000;&quot;&gt;Ｃ&lt;/span&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#ff0000;&quot;&gt;♢&lt;/span&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#000000;&quot;&gt;ＮＤＵＭ&lt;/span&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#ffee00;&quot;&gt;Λ&lt;/span&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#000000;&quot;&gt;Ｓ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="wallpaper.qrc"/>
 </resources>
 <connections/>
</ui>
